src/core/DisplayGroup.cpp

```diff
@@ -176,6 +200,13 @@ void DisplayGroup::setBackgroundContent( ContentPtr content )
     sendDisplayGroup();
 }
 
+void DisplayGroup::setShowWindowTitles( bool set )
+{
+    showWindowTitles_ = set;
```

Comment:
do:
if( showWindowTitles_ == set )
    return;
to save a sendDisplayGroup if there is no change